Watch Extension/AddFeedingView.swift

```diff
@@ -108,6 +89,7 @@ struct AddFeedingView: View {
 
 struct ContentView_Previews: PreviewProvider {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/443d289c50906f54d351235f32088d9c382abf56/Watch Extension/AddFeedingView.swift#L90"></span>Type name should only contain alphanumeric characters: ‘ContentView_Previews’<br />
<code>type_name</code> <code>AddFeedingView.swift:90</code></td>
    </tr>
  </tbody>
</table>